api/HTMLMediaElement.json

```diff
@@ -2209,7 +2209,7 @@
               "version_added": "43"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "43"
```

Comment:
Most of the data being mirrored here is incorrect, see https://github.com/mdn/browser-compat-data/issues/7843.

Can you prepare a separate PR updating only the "43" values and mirroring from the new correct data?

Reply:
Okay, this is odd -- the collector isn't changing the value for this feature, and yet running the test in Chrome 42 shows this is clearly supported beforehand.  Yikes...

I've been noticing some odd issues with updates to Chrome Desktop's data recently.  :/

Reply:
Can you file a collector issue about that? It's probably due to one of the TODOs in the updater code...